### ISSUE TITLE
`pipenv --rm` does not remove `src` folder

### ISSUE THREAD

**Issue by manihamidi** [pypa/pipenv#1516]
`pipenv --rm` does not remove everything under the `/user/.virtualenvs/<repo>-<hash>/` folder. It removes everything except the `src` folder. 

This may be by design, but resulted in some unexpected errors when I did `pipenv --rm` and then `pipenv install` (to start 'afresh' and make sure everything is in order!):

```
Error:  An error occurred while installing -e git+https://github.com/MyCompany/MyTool@v2.0.1#egg=my_tool-2.0.1!
Command "git config --get-regexp remote\..*\.url" failed with error code 1 in C:\Users\Mani\.virtualenvs\MyTool-Vb5JMSd4\src\my-tool
```

##### Describe your environment

1. OS Type windows 7
1. Python version: `$ python -V` 2.7.12
1. Pipenv version: `$ pipenv --version` 10.1.0

##### Expected result

Would have expected it to just re-install everything (including a clone/install off of github). Manually removing the `src/` folder, before attempting `pipenv install` *does* achieve this.

**Comment by uranusjr** [comment]
`--rm` does not do anything fancy, but simply attempts to remove the whole virtualenv directory. What you see is like caused by some underlying error. Pipenv (which uses `shutil.rmtree` underneath) simply ignores any failures it encountered during the process.

Maybe it would be more user-friendly to use the `onerror` handler in this situation, and omit a warning instead?

**Comment by altendky** [comment]
@manihamidi maybe fixed by #1927.
